core/modules/utils/dom/modal.js

```diff
@@ -58,6 +58,9 @@ Modal.prototype.display = function(title,options) {
 	this.adjustPageClass();
 	// Add classes
 	$tw.utils.addClass(wrapper,"tc-modal-wrapper");
+	if(tiddler && tiddler.fields && tiddler.fields.class) {
```

Comment:
There's a test for "tiddler" being falsey on line 40, so you can avoid checking that. I notice there's the same issue on line 107, if you'd like to kindly address that at the same time.

Reply:
Resolved.